data/json/mapgen/house/house_vacant.json

```diff
@@ -1575,7 +1593,15 @@
         { "chance": 50, "item": "2x4", "repeat": 1, "x": 12, "y": 16 },
         { "chance": 50, "item": "2x4", "repeat": 1, "x": 17, "y": 17 }
       ],
-      "furniture": { "#": "f_null", ".": "f_null", "_": "f_null", "s": "f_null", "t": "f_null", "|": "f_null" },
+      "furniture": {
+        "#": "f_null",
+        ".": "f_null",
+        "_": "f_null",
+        "P": "f_mailbox",
+        "s": "f_null",
+        "t": "f_null",
+        "|": "f_null"
+      },
```

Comment:
Should be linted and look like earlier:

![image](https://user-images.githubusercontent.com/16213433/45518588-c2838780-b7ba-11e8-9aaa-9c7fee094fee.png)